tst/testlibgap/wsload.c

```diff
@@ -4,7 +4,14 @@
 #include "common.h"
 int main(int argc, char ** argv)
 {
-    GAP_Initialize(argc, argv, environ, 0L, 0L);
+    char *args[50];
+    char lpar[3] = "-L";
+    char wsname[16] = "/tmp/libgap.ws"; /* the name must match the one used in wscreate.c */
+    memcpy(args, argv, argc*sizeof(char*));
+    args[argc] = lpar;
+    args[argc+1] = wsname;
+    args[argc+2] = NULL;
```

Comment:
I think (but have no  verified) that you could replace the above hackery by just this:
```C
  SyRestoring = wsname;
```

Reply:
Did you try this?

Reply:
for starters, this does not compile, and I wonder whether that `SyRestoring` should be exported.

Reply:
Fair enough